Formula/rgbds.rb

```diff
@@ -21,24 +21,20 @@ class Rgbds < Formula
   end
 
   depends_on "bison" => :build
```

Comment:
Is this still needed?

Reply:
Yes, we're still generating the assembler's parser from a Bison grammar. And soon enough, the linker's linker script parser too :)

Reply:
It looks like it might be able to work with macOS-provided Bison, though? https://github.com/gbdev/rgbds/blob/28737d5778f05bd554e595360c1f94d21328d15d/src/CMakeLists.txt#L17

Reply:
Unless they started providing at least Bison 3.0.0, no. We've had reports of build failures on macOS, which [prompted us to add this](https://github.com/gbdev/rgbds/blob/28737d5778f05bd554e595360c1f94d21328d15d/.github/workflows/testing.yml#L32-L38).

Reply:
The `find_package` call should probably specify the minimum required version then.

Reply:
I'll give you that we're being sloppy on that.